src/Refactoring-Transformations-Tests/RBPullUpMethodParametrizedTest.class.st

```diff
@@ -263,19 +263,21 @@ RBPullUpMethodParametrizedTest >> testPullUpReferencesInstVar [
 	class := model classNamed: ('RBTransformation', 'RuleTestData1') asSymbol.
 	refactoring := self createRefactoringWithModel: model
 		andArguments: { #(#foo ) . class }.
-	[ self executeRefactoring: refactoring ] valueSupplyingAnswer: true.
+	self proceedThroughWarning: [ self executeRefactoring: refactoring  ].
 	superClass := model classNamed: ('RBFooLint', 'RuleTestData1') asSymbol.
 	self assert: (superClass parseTreeForSelector: #foo)
 				equals: (self parseMethod: 'foo ^ foo').
-	self assert: (superClass directlyDefinesInstanceVariable: #foo).
 	self deny: (class directlyDefinesMethod: #foo).
-	self deny: (class directlyDefinesInstanceVariable: #foo)
+	self assert: (class directlyDefinesInstanceVariable: #foo)
 ]
 
 { #category : 'failure tests' }
 RBPullUpMethodParametrizedTest >> testWarnDoesntPullUpReferencesInstVar [
 
 	| refactoring class |
+	rbClass = RBPullUpMethodTransformation ifTrue: [ " RBPullUpMethodTransformation shouldn't warn when instance variables are referenced "
+		self skip ].
+	
 	class := model classNamed:
 		         ('RBTransformation' , 'RuleTestData1') asSymbol.
 	refactoring := self
```

Comment:
This is a ok, but we were starting to separate tests for refactoring and transformations. Transformations tests are parametrized with refactorings as well, and refactorings only have their own tests that are concerned with breaking changes, so in a next PR it makes sense to move this to a new test class for refactoring.

Reply:
I created a new [issue](https://github.com/pharo-project/pharo/issues/17157) to move refactoring tests to a new class